builtin/common/filterlist.lua

```diff
@@ -198,8 +195,7 @@ function filterlist.process(self)
 	end
 
 	if self.m_sort_list[self.m_sortmode] ~= nil and
-		type(self.m_sort_list[self.m_sortmode]) == "function" then
-
+			type(self.m_sort_list[self.m_sortmode]) == "function" then
```

Comment:
You can shorten this code a bit: The `self.m_sort_list[self.m_sortmode] ~= nil` is redundant because `type` returns `"nil"` if nil is passed.